src/tracing/node_trace_buffer.h

```diff
@@ -75,7 +75,7 @@ class NodeTraceBuffer : public TraceBuffer {
   // Used to wait until async handles have been closed.
   ConditionVariable exit_cond_;
   std::unique_ptr<NodeTraceWriter> trace_writer_;
-  // TODO: Change std::atomic to something less contentious.
+  // TODO(matthewloring): Change std::atomic to something less contentious.
```

Comment:
Not required as part of this change but this TODO is no longer necessary. We thought this might cause issues when the initial PR was reviewed but it ended up not being problematic.